dockerfiles/ci-unified/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM docker.io/library/debian:bullseye-20230522-slim
+FROM docker.io/library/debian:bookworm-20231120-slim
 
 
 ### meta ###
 
-ARG DISTRO_CODENAME="bullseye"
+ARG DISTRO_CODENAME="bookworm"
 ARG RESTIC_VERSION="0.16.1"
-ARG RUST_STABLE_VERSION="1.73.0"
+ARG RUST_STABLE_VERSION="1.74.0"
```

Comment:
should you change that as part of this PR?

Reply:
It's a cosmetic change actually. We specify the rust version using CI variables in gitlab. Currently polkadot-sdk uses rust 1.74 in CI